WordPress/src/main/java/org/wordpress/android/ui/people/utils/PeopleUtils.java

```diff
@@ -46,8 +49,13 @@ public void onErrorResponse(VolleyError volleyError) {
             }
         };
 
+        Map<String, String> params = new HashMap<>();
+        params.put("number", Integer.toString(PeopleUtils.FETCH_USERS_LIMIT));
```

Comment:
Thanks for the heads up @jleandroperez. I am currently working on the follower section and I'll make sure we use the correct parameters for this. We currently use the default values, so probably that's why I didn't notice the issue before and just used what the documentation said here: https://developer.wordpress.com/docs/api/1.1/get/sites/%24site/users/